apps/frontend/src/components/call/CallListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { Draggable } from '@hello-pangea/dnd';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { useTheme } from '@mui/material/styles';
+import React from 'react';
+
+import { Call } from 'generated/sdk';
+
+export type DraggableListItemProps = {
+  item: Call;
+  index: number;
+};
+
+export default function DraggableListItem({
+  item,
+  index,
+}: DraggableListItemProps) {
+  const theme = useTheme();
+
+  return (
+    <Draggable draggableId={item.id.toString()} index={index}>
+      {(provided, snapshot) => (
+        <ListItem
+          data-cy={'drag-item'}
```

Comment:
If this is for this component maybe prefix with something like `call-list-drag-item`.